cmd/fyne/commands/release.go

```diff
@@ -13,21 +11,131 @@ import (
 	"fyne.io/fyne/v2/cmd/fyne/internal/mobile"
 	"fyne.io/fyne/v2/cmd/fyne/internal/templates"
 	"fyne.io/fyne/v2/cmd/fyne/internal/util"
+	"github.com/urfave/cli/v2"
 )
 
-var (
-	macAppStoreCategories = []string{
-		"business", "developer-tools", "education", "entertainment", "finance", "games", "action-games",
-		"adventure-games", "arcade-games", "board-games", "card-games", "casino-games", "dice-games",
-		"educational-games", "family-games", "kids-games", "music-games", "puzzle-games", "racing-games",
-		"role-playing-games", "simulation-games", "sports-games", "strategy-games", "trivia-games", "word-games",
-		"graphics-design", "healthcare-fitness", "lifestyle", "medical", "music", "news", "photography",
-		"productivity", "reference", "social-networking", "sports", "travel", "utilities", "video", "weather",
-	}
-)
+var macAppStoreCategories = []string{
```

Comment:
These are no longer current as of February 5th, 2020 with the universal platform:
https://developer.apple.com/news/?id=02052020a

Reply:
Pretty sure that that isn't in the scope of this PR. Better to open up an issue or PR for that in the future.